wordpress-mobile/WordPress-iOS
# [Stats Refresh] Insights Latest Post Summary: update design

Fixes #11284

This updates Insights Latest Post Summary to match the new design:
- Move “views” to top and use full number
- Move “likes” and “comments” to bottom as table rows
- The Views number now abbreviates after 99,999.

To test:

---
- On a site that has a post with views/comments/likes, go to Insights > Latest Post Summary.
- Verify the view is now:

<img width="375" alt="lps" src="https://user-images.githubusercontent.com/1816888/54459400-37188e00-472c-11e9-8793-60fe5a5da0aa.png">

---
- On a site that has a post with no views/comments/likes, go to Insights > Latest Post Summary.
- Verify the view is still: (the point being there should be no change here)

<img width="350" alt="no_data" src="https://user-images.githubusercontent.com/1816888/54459301-ef920200-472b-11e9-9c5f-b8b9aea01c47.png">

---
- On a site with a more than 99,999 views, go to Insights > Latest Post Summary.
- Verify the number is abbreviated.
  - Note: to test this, in `LatestPostSummaryCell:configure`, you can modify the value used in this line:
`viewsDataLabel.text = lastPostInsight.viewsCount.abbreviatedString(forHeroNumber: true)`

<img width="350" alt="abbr" src="https://user-images.githubusercontent.com/1816888/54459654-e6edfb80-472c-11e9-8892-cbd4d1225e7d.png">
